compiler/src/dmd/attrib.d

```diff
@@ -520,22 +520,23 @@ extern (C++) final class CPPMangleDeclaration : AttribDeclaration
  */
 extern (C++) final class CPPNamespaceDeclaration : AttribDeclaration
 {
+nothrow:
     /// CTFE-able expression, resolving to `TupleExp` or `StringExp`
     Expression exp;
 
-    extern (D) this(const ref Loc loc, Identifier ident, Dsymbols* decl) @safe
+    extern (D) this(const ref Loc loc, Identifier ident, Dsymbols* decl) nothrow @safe
```

Comment:
I'd have thought that most the time all you need is `nothrow:` at the top of the class/struct member list, and not adding `nothrow` to every single member individually.

Reply:
Yes. If that way is considered better style it certainly makes a smaller diff.

Reply:
Changed to do things this way within class/struct declarations but not at the tops of modules.